libkbfs/keybase_service_util.go

```diff
@@ -62,6 +62,19 @@ func serviceLoggedIn(ctx context.Context, config Config, name string,
 		}
 	}
 
+	if adminFeatureList[session.UID] {
+		log.CDebugf(ctx, "Enabling a dir op batch size of %d",
+			bgFlushDirOpBatchSizeDefault)
+		// NOTE: This overrides any command-line parameter.  It only
+		// matters until we un-feature-flag this, I think it's ok for
+		// now.
+		config.SetBGFlushDirOpBatchSize(bgFlushDirOpBatchSizeDefault)
```

Comment:
Only when an admin logs in _after_ KBFS initializes.  I know that's not ideal, but it's temporary until this is default for everyone.  Open to other ideas -- I just didn't want to plumb an extra config field here for the original value of the command line flag.